packages/client/src/components/ratings/FutureProofRatingView/FutureProofRatingView.tsx

```diff
@@ -1,15 +1,25 @@
 import CertificatesList from '@components/ratings/CertificateList'
 import CircularRatingIndicator from '@components/ratings/CircularRatingIndicator'
 import RatingBreakdownItems from '@components/ratings/RatingBreakdownItems'
-import { BusinessCertificate, DisplayableBusiness, Rating } from '@futureproof/typings'
+import RatingBreakdownItemsAlt from '@components/ratings/RatingBreakdownItems/RatingBreakdownItemsAlt'
+import { BusinessCertificate, DisplayableBusiness, Business, Rating } from '@futureproof/typings'
 import React from 'react'
-import { SafeAreaView, ScrollView, StyleSheet, Text, View } from 'react-native'
+import { SafeAreaView, ScrollView, StyleSheet, Text, View, Image, Dimensions } from 'react-native'
+import { StackScreenProps } from '@react-navigation/stack'
 
 type ComponentProps = Pick<
```

Comment:
This type isn't used anymore - do we still need it?

Reply:
Ok.